tests/e2e/pact/specs/live/process.successes.spec.ts

```diff
@@ -34,14 +34,6 @@ const orderUpdatePactGroups = [
     name: "secondarycare-community-acute-cancel",
     cases: TestResources.processSecondaryCareCommunityAcuteOrderUpdateCases
   },
-  {
-    name: "secondarycare-community-repeatdispensing-cancel",
```

Comment:
how come these are being removed?

Reply:
we don't have any example cancelations for these groups, only secondarycare-community-acute-cancel, and primarycare-cancel